guides/source/5_0_release_notes.md

```diff
@@ -37,9 +37,20 @@ Major Features
 --------------
 
 ### Action Cable
-[Pull Request](https://github.com/rails/rails/pull/22586)
 
-ToDo...
+Action Cable is a new framework in Rails 5. It seamlessly integrates
```

Comment:
It's a framework. Our overview, etc refer it as framework, see below server-side framework, javascript framework, etc.
I am also basing this on AJ release notes, which was referred to as a framework.